.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "editor.codeActionsOnSave": { "source.fixAll.eslint": true },
+  "[typescript]": {
+    "editor.defaultFormatter": null // disable prettier if present. Let eslint handle it.
+  }
+}
```

Comment:
Not sure if we want this, but my local prettier rules were formatting the file overriding the required eslint rules established in the repo.